h2/src/test/org/h2/test/db/TestRunscript.java

```diff
@@ -51,6 +51,7 @@ public void test() throws Exception {
         testCancelScript();
         testEncoding();
         testClobPrimaryKey();
+        testComment();
```

Comment:
Please, move this test to `createTable.sql`. `SCRIPT TABLE TEST1` output is not too big and I don't see any reason to use custom checks for presence of substrings.

Reply:
How to skip "CREATE USER " row comparison in createTable.sql ?
 Every time i run test salt and hash are different.
Example:
SCRIPT TABLE TEST1;
> SCRIPT
> ----------------------------------------------------------------------------------------------------------------------
> -- 0 +/- SELECT COUNT(*) FROM PUBLIC.TEST1;
> ALTER TABLE "PUBLIC"."TEST1" ADD CONSTRAINT "PUBLIC"."CONSTRAINT_4" PRIMARY KEY("ID");
> CREATE CACHED TABLE "PUBLIC"."TEST1"( "ID" BIGINT NOT NULL COMMENT 'COMMENT1', "FIELD_NAME" VARCHAR(100) NOT NULL COMMENT 'COMMENT2' );
> CREATE USER IF NOT EXISTS "SA" SALT '7f0c86261a0b7222' HASH '920138111dd8344f1d4520e6f99a58040e852b7361d552397f1cd4f18a6debc1' ADMIN;
> rows: 4

Reply:
Try
```SQL
SCRIPT NODATA NOPASSWORDS NOSETTINGS TABLE TEST
```
It should create reproducible results.

Reply:
`NODATA` is not really required, of course, because your table is empty.